css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+p,li,dt,dd,td,th,details,button,form,a{font-family: Raleway;}
```

Comment:
Fair enough, Bootstrap theme applies the font-family attribute on the Body, so it always goes inherits back, but happy to apply here again.